ceremonies/src/tests.rs

```diff
@@ -1009,48 +954,7 @@ fn early_rewards_works() {
 }
 
 #[test]
-fn early_rewards_with_new_attest_attendees_extrinsic_works() {
```

Comment:
this entire test is redundant since we removed the old extrinsic and adjusted the other test to cover this